### ISSUE TITLE
plaidml errors - objects treated differently

### ISSUE THREAD

**Issue by jimr1603** [rstudio/keras#424]
`library(keras)
model <- application_vgg16(
  weights = "imagenet", 
  include_top = FALSE
)
layer_name <- "block3_conv1"
filter_index <- 1
layer_output <- get_layer(model, layer_name)$output`

This works, and:

> class(layer_output)
[1] "tensorflow.tensor"                           "tensorflow.python.framework.ops.Tensor"     
[3] "tensorflow.python.framework.ops._TensorLike" "python.builtin.object"    

However, changing backend to plaidml gives

> loss <- k_mean(layer_output[,,,filter_index])
Error in layer_output[, , , filter_index] : 
  object of type 'environment' is not subsettable

> class(layer_output)
[1] "plaidml.tile.Value"           "plaidml.tile._ShapelessValue" "python.builtin.object"       

I never got CUDA running, so my tensorflow example is on CPU.
